Add functionality to set labels as html

chivandikwa opened the pull request:
Added functionality to allow for html labels.

Related to issue: https://github.com/Rubjerg/Graphviz.NetWrapper/issues/14

Chiel92 commented:
Thanks, looks good. It seems that the CI suddenly broke with no code changes -.- I'm gonna have to fix that first before I can merge

chivandikwa commented:
> Thanks, looks good. It seems that the CI suddenly broke with no code changes -.- I'm gonna have to fix that first before I can merge

Any ETA on this? I really depend on this change for my use cases of your awesome wrapper.

Chiel92 commented on Rubjerg.Graphviz/ForeignFunctionInterface.cs:
```diff
@@ -143,7 +143,28 @@ public static void Agset(IntPtr obj, string name, string value)
             }
+        }   
+        
+        public static void AgreLabelNode(IntPtr obj, string value)
```
I think a name like `AddLabel` would be clearer. The Ag prefix is only taken over from functions that are more or less directly exposed (modulo the mutex). And this function would work for any object, not just nodes, correct?

chivandikwa replied:
Yes, you are right. Agreed and updated

Chiel92 commented:
> > Thanks, looks good. It seems that the CI suddenly broke with no code changes -.- I'm gonna have to fix that first before I can merge
> 
> Any ETA on this? I really depend on this change for my use cases of your awesome wrapper.

The CI doesn't seem to have a trivial fix, but since you are only exposing new functionality, nothing existing can break, we don't have to wait till I fix the CI.

Chiel92 commented:
Ok I fixed appveyor. If you merge master into your branch the build should be green.

chivandikwa commented:
> Ok I fixed appveyor. If you merge master into your branch the build should be green.

My apologies for going silent, somehow missed these notifications.

I have merge as instructed and addressed one change request.

Chiel92 commented on Rubjerg.Graphviz/ForeignFunctionInterface.cs:
```diff
@@ -137,13 +138,61 @@ public static void Agattr(IntPtr graph, int type, string name, string deflt)
+            }
+        }   
+        
+        private static void AgsetLabel(IntPtr obj, string value)
```
This method seems obsolete to me, as it does the same `AddLabel`, correct?

Chiel92 reviewed (changes requested):
Thanks for your update @chivandikwa. I've got another small comment in response.

Chiel92 commented on Rubjerg.Graphviz/ForeignFunctionInterface.cs:
```diff
@@ -137,13 +138,42 @@ public static void Agattr(IntPtr graph, int type, string name, string deflt)
+        {
+            lock (Mutex)
+            {
+                const string labelKey = "label";
```
This should still be removed in favour of the member variable `LabelKey`
